CMakeLists.txt

```diff
@@ -188,39 +188,42 @@ if(TARGET hdMtlx)
     endif()
 endif()
 
-if(MAYA_APP_VERSION VERSION_GREATER 2024)
-    # First look for Qt6 in the Maya devkit.
-    # The Qt6 archive in the Maya devkit contains everything needed for the normal cmake find_package.
-    set(CMAKE_PREFIX_PATH "${MAYA_DEVKIT_LOCATION}/Qt")
-    find_package(Qt6 6.5 COMPONENTS Core Gui Widgets QUIET)
-endif()
-if (Qt6_FOUND)
-    message(STATUS "Found Qt6 in Maya devkit, building with Qt ${Qt6_VERSION} features enabled.")
-    set(Qt_FOUND TRUE)
-else()
-    # If we didn't find Qt6 in Maya devkit, search again, but for Qt5 this time.
-    # Minimum version required is 5.15 (Maya 2022/2023/2024).
-    # This will find Qt in the Maya devkit using a custom find package.
-    # So the version will match Maya we are building for.
-    find_package(Maya_Qt 5.15 COMPONENTS Core Gui Widgets QUIET)
-    if (Maya_Qt_FOUND)
-        message(STATUS "Found Qt5 in Maya devkit, building with Qt ${MAYA_QT_VERSION} features enabled.")
-        set(Qt_FOUND TRUE)
-    endif()
-endif()
-    
-if(NOT Qt_FOUND)
-    message(SEND_ERROR "Could not find Qt in Maya devkit directory: ${MAYA_DEVKIT_LOCATION}.")
+# Only need Qt when building MayaUsd (i.e. Qt not needed for UsdUfe standalone build).
+if(BUILD_MAYAUSD_LIBRARY)
```

Comment:
If you "Hide whitespace" changes you'll see that I just added this if case here and put all the Qt stuff inside of it. When building standalone usdUfe we don't need Qt.